src/Stores/Promises/__tests__/selectors.test.js

```diff
@@ -0,0 +1,108 @@
+/* @flow */
+
+import { promisesSelector } from '../selectors';
+import { initialState, politicians, promises } from '../../../utils/testFixtures';
+
+describe('Promises selector', () => {
+  it('loads initial state', () => {
+    expect(promisesSelector(initialState)).toEqual(null);
+  });
+
+  it('Returns null if politicians arent loaded', () => {
+    const loadedState = {
+      ...initialState,
+      promises: {
+        ...initialState.promises,
+        promises: [promises[0]],
+      },
+    };
+    expect(promisesSelector(loadedState)).toEqual(null);
+  });
+
+  it('Returns promises', () => {
+    // TODO use the enriched politician object
```

Comment:
Not sure what that means exactly but might be nicer to move the test data to specific files? So they dont pollute the tests so much and can be reused :)